source/persistence/MessageController.cpp

```diff
@@ -342,8 +347,8 @@ QString MessageController::getRessourceForMsgId(const QString& msgId)
 
     QSqlQuery query(*(database_->getPointer()));
 
```

Comment:
This still is the wrong query to return the jid resource. Maybe there was mistake during your rollback?

Reply:
Sorry. I fixed it.